Web/starter_project_vanguard/pages/profile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link'
+import React, { useState } from 'react'
+
+import AccountSetting from '@/components/profile/AccountSetting'
+import PersonalInfo from '@/components/profile/PersonalInfo'
+
+const Profile: React.FC = () => {
+  const [tab, setTab] = useState<number>(0)
+  const handleTabChange = (tab: number) => {
+    setTab(tab)
+  }
+
+  return (
+    <div className="px-10">
+      <h1 className="font-medium font-poppins text-[40px]">Profile</h1>
```

Comment:
[nit] can you make the font size of this section somewhat proportional to the rest of the page?